### DISCUSSION TITLE
Management URIs protocol

### DISCUSSION THREAD

**Discussion by vitalie-cracan** [Azure/azure-functions-durable-extension#2113] [Q&A] [answered]
I am starting an orchestration with a Durable Functions HTTP starter, my Function App is set to HTTPS only. However the returned instance manangement URIs use the http protocol, not https. Why? How can I make sure these URIs use https?

`return client.create_check_status_response(req, instance_id)`

Thank you.

**Comment by cgillum** [comment]
@davidmrdavid do you know the answer to this (this is using the Durable Functions Python SDK)?

**Reply by davidmrdavid** [reply]
Thanks for the ping @cgillum.
I have personally not tested this scenario, but I've created a [tracking ticket](https://github.com/Azure/azure-functions-durable-python/issues/355) to look into it.

Thank you @en-vcracan for the pointer to [this](https://github.com/Azure/azure-functions-host/issues/6602) issue. It will be really helpful in our investigation

**Comment by vitalie-cracan** [comment, marked as answer]
I think I am experiencing this issue: https://github.com/Azure/azure-functions-host/issues/6602
